src/nebula-hooks/use-context-menu.ts

```diff
@@ -1,10 +1,9 @@
 // @ts-ignore  TODO: remove tag when onContextMenu is declared in nebula/stardust
 import { onContextMenu } from '@nebula.js/stardust';
 import { copyCellValue } from '../table/utils/accessibility-utils';
-import { Menu } from '../types';
 
 export default function useContextMenu(areBasicFeaturesEnabled: boolean) {
-  onContextMenu((menu: Menu, event: any) => {
+  onContextMenu((menu: any, event: any) => {
```

Comment:
As we decided to set menu type to `any` I should change this which was missed in the original PR, It's a small change so I fixed through this PR.

Reply:
sure :)